projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -1512,4 +1512,14 @@ public function can_blaze() {
 	public function get_wpcom_site_setup() {
 		return get_option( 'wpcom_site_setup' );
 	}
+
+	/**
+	 * Return true if the site is a commercial site. If not determined yet, it returns null.
```

Comment:
To clarify that we have 3 different states for `is_commercial`.
- True = the site is commercial
- False = the site is not commercial
- NULL = the commercial status is not yet defined

I tested the code of this function, and it gets all of these states correctly, but the PHPDoc should be updated.

Reply:
Thanks for the review @sotirispl 👍 

Updated PHPDoc in 0979f04f10.